seatunnel-connectors-v2/pom.xml

```diff
@@ -56,5 +56,16 @@
             <artifactId>junit-jupiter-params</artifactId>
         </dependency>
     </dependencies>
+    <build>
```

Comment:
If skip alway is true, maybe you can remove it.

Reply:
We have to statement to do that.

Reply:
Got it